File: src/main/java/parser/ArithmeticTokenType.java
package parser;

public enum ArithmeticTokenType implements IArithmeticTokenType {
    PLUS,
    MINUS,
    STAR,
    SLASH;

    @Override
    public String toString() {
        switch (this) {
            case PLUS:
                return "+";
            case MINUS:
                return "-";
            case STAR:
                return "*";
            case SLASH:
                return "/";
        }

        return "";
    }
}

File: src/main/java/parser/ArithmeticTokenType.kt
package parser

enum class ArithmeticTokenType : IArithmeticTokenType {
    PLUS,
    MINUS,
    STAR,
    SLASH;

    override fun toString(): String {
        return when (this) {
            PLUS -> "+"
            MINUS -> "-"
            STAR -> "*"
            SLASH -> "/"
        }
    }
}
